### PR TITLE
Added About page

### PR DESCRIPTION
**Pull request by Pdbz199**
Lots of changes here, sorry about that. I removed a bunch of stuff that was remaining from Kotlin and added the About page. I thought it was better to have the page be an activity itself, but please let me know your thoughts. Thanks.

### CODE REVIEW

**Comment by Pdbz199** [comment]
Screenshot_20190428-095431_ithaca-transit
Screenshot_20190428-095442_ithaca-transit

File: .idea/gradle.xml [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
+        <compositeConfiguration>
```
Comment (ningning621) [inline .idea/gradle.xml]: was it necessary to push this file?

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                        }
+                        mAboutView.closeDrawer(mAboutMenu);
+                        return true;
+                    }
+                }
+        );
+    }
+
+    private void makeHereClickable() {
+        SpannableString ss = new SpannableString("An Engineering Project Team at Cornell University. View more of our apps here.");
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java]: put string inside `strings.xml`

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        ClickableSpan clickableSpan = new ClickableSpan() {
+            @Override
+            public void onClick(View textView) {
+                Intent intent = new Intent(Intent.ACTION_VIEW);
+                intent.setData(Uri.parse(
+                        "https://play.google.com/store/apps/developer?id=Cornell+AppDev"));
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java]: `string.xml` pls

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        ss.setSpan(clickableSpan, ss.toString().indexOf("here"), ss.toString().indexOf("here") + 4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+
+        TextView textView = (TextView) findViewById(R.id.about_proj_team);
+        textView.setText(ss);
+        textView.setMovementMethod(LinkMovementMethod.getInstance());
+        textView.setHighlightColor(Color.TRANSPARENT);
+    }
+
+    private void makeButtonsClickable() {
+//        onboarding.setOnClickListener(new View.OnClickListener() {
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java]: leave a TODO

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+            @Override
+            public void onClick(View v) {
+                sendFeedback();
+            }
+        });
+
+        website.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                String url = "https://www.cornellappdev.com/";
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/AboutActivity.java]: put inside `string.xml`

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/DirectionsListAdapter.java [hunk 6]
```diff
@@ -75,7 +75,7 @@ public TextView getDirectionDistance() {
             return directionDistance;
         }
 
-        public TextAdapterViewHolder(@NotNull View itemView) {
+        public TextAdapterViewHolder(View itemView) {
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/DirectionsListAdapter.java:78]: Not saying that this is wrong but why was this changed?

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java [hunk 7]
```diff
@@ -14,6 +18,7 @@
 import android.support.v4.widget.DrawerLayout;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.RecyclerView;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java]: delete

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java [hunk 8]
```diff
@@ -152,13 +156,43 @@ public void onDrawerStateChanged(int newState) {}
...
+                    }
                     mHomeView.closeDrawer(mHomeMenu);
                     return true;
                 }
             }
         );
     }
 
+    private void sendFeedback() {
+        Intent emailIntent = new Intent(android.content.Intent.ACTION_SEND);
```
Comment (ningning621) [inline app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java:169]: oooo nice use of intents

**Review by ningning621 — CHANGES_REQUESTED** [review]
left some minor comments for changes but overall looks good

File: app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/DirectionsListAdapter.java (hunk 6 above)
Comment (Pdbz199) [reply app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/DirectionsListAdapter.java:78]: I'm not sure why, but it wasn't building on either of my computers

**Review by ningning621 — APPROVED** [review]

**Comment by ningning621** [comment]
lgtm make sure to lint and rebase!

**Comment by ningning621** [comment]
Addresses #16
